submissions/TaniaBondarenko/frogger_game/app.js

```diff
@@ -2,62 +2,76 @@ const borders = {
     rightBorder: 400,
     leftBorder: 5,
     upBorder: 410,
-    downBorder: 70
+    downBorder: 5
 };
+const someGap = 10;
 
-const Enemy = function (x, y) {
-    this.sprite = 'images/enemy-bug.png';
-    this.x = x;
-    this.y = y;
-    this.speed = 75;
-
-    this.render = function () {
+class GameSprite {
+    constructor (x, y,sprite)  {
+        this.sprite = sprite;
+        this.x = x;
+        this.y = y;
+    }
+    render = function () {
         ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
     };
+}
+
+class Enemy extends GameSprite {
+    constructor(x, y, sprite = "images/enemy-bug.png", speed = "75") {
+        super(x, y, sprite);
+        this.x = x;
+        this.y = y;
+        this.speed = speed;
+        this.sprite = sprite;
+    }
+    render(){};
```

Comment:
we already have 'render()' method defined in the parent class 'GameSprite'.
it's better to remove this line, because it redefines 'render()' to an empty function here.

Reply:
okay